aws/resource_aws_glue_catalog_table_test.go

```diff
@@ -31,15 +31,15 @@ func TestAccAWSGlueCatalogTable_recreates(t *testing.T) {
 				PreConfig: func() {
```

Comment:
Created followup issue for this older disappears testing style: https://github.com/terraform-providers/terraform-provider-aws/issues/13820